pkg/volume/plugins.go

```diff
@@ -216,6 +216,20 @@ type ExpandableVolumePlugin interface {
 	RequiresFSResize() bool
 }
 
+// VolumePluginWithAttachLimits is an extended interface of VolumePlugin that restricts number of
+// volumes that can be attached to a node.
+type VolumePluginWithAttachLimits interface {
+	VolumePlugin
+	// Return number of volumes attachable on a node from the plugin
```

Comment:
* Please add some description what is the return map - what's the key and what's value.
* I'd emphasize that it's called by kubelet and it returns limit for the node where this function is called.